core/tests/swfs/avm2/bytearray/test.as

```diff
@@ -0,0 +1,38 @@
+import flash.utils.ByteArray;
```

Comment:
It seems that this Action Script code is not being loaded from the FLA file.

Change file name from `test.as` to `Test.as` and then create a Test Class as shown below.
```as
package {
    public class Test {
        public function Test() {}
    }
}

import flash.utils.ByteArray;
import flash.utils.Endian;
var test = new ByteArray();
test.writeUTFBytes("HELLO TEST");
test.position = 0;
trace(test.readByte());
...snip...
```

Modify the code in the FLA file as follows:
```as
import Test;
new Test();
```